src/wizards/tokenSwapDealWizard/components/tokenDetails/tokenDetails.html

```diff
@@ -0,0 +1,184 @@
+<template>
+  <pcard class="${!valid ? 'invalidTokenDetails' : '' }">
+    <div if.bind="viewMode === 'edit'">
+
+      <pform-input
+        data-test="proposal-token-address-field"
+        helper-message.bind="tokenInfoLoading ? 'Searching token address details...': ''"
+        input-reference.bind="addressInput"
+        label="Token Address"
+        label-info="Some info here"
+      >
+        <pinput-text
+          input.trigger="getTokenInfo(token.address)"
+          placeholder="0x..."
+          validation-state.bind="tokenInfoLoading? 'validating': addressInput.validationState !== 'validating' ? addressInput.validationState : undefined"
+          value.bind="token.address & validate"
+          view-model.ref="addressInput"
+        ></pinput-text>
+      </pform-input>
+      <div class="detailsTable" if.bind="showTokenDetails">
+        <div class="detailsCell">
+          <p>Name</p>
+          <p if.bind="!tokenDetailsNotFound.name">${token.name}</p>
+          <pinput-text if.bind="tokenDetailsNotFound.name" value.bind="token.name"></pinput-text>
+        </div>
+        <div class="detailsCell">
+
+          <p>Symbol</p>
+          <p if.bind="!tokenDetailsNotFound.symbol">${token.symbol}</p>
+          <pinput-text if.bind="tokenDetailsNotFound.symbol" value.bind="token.symbol"></pinput-text>
+        </div>
+        <div class="detailsCell">
+          <p>Decimals</p>
+          <p if.bind="!tokenDetailsNotFound.decimals">${token.decimals}</p>
+          <pinput-numeric if.bind="tokenDetailsNotFound.decimals" not-wei value.bind="token.decimals"></pinput-numeric>
+        </div>
+        <div class="detailsCell">
+          <p>
+            Logo
+            <img error.trigger="logoLoaded(false)" height="24" if.bind="validTokenLogoURI"
+                 load.trigger="logoLoaded(true)"
+                 src.bind="token.logoURI">
+          </p>
+          <pinput-text input.trigger="checkURL()" value.bind="token.logoURI"></pinput-text>
+          <span class="feedbackMessage ${!validTokenLogoURI ? 'feedbackMessageVisible' : ''} errorMessage">
+          No valid image found at the provided logo URL
+        </span>
+        </div>
+      </div>
+      <pform-input data-test="proposal-token-amount-field"
+                   input-reference.bind="amountInput"
+                   label="Amount"
+                   label-description="You are able to edit the amount of tokens to be swapped before the execution phase."
+      >
+        <pinput-numeric
```

Comment:
note you can just say `not-wei`.  I'm not sure that `.bind="true"` is even valid since `true` isn't a view-model instance variable.  Should be `not-wei="true"`.  But just plain `not-wei` is nicest.